examples/getting_started/quickstart.ipynb

```diff
@@ -0,0 +1,626 @@
+    "    print(f\"The capital of {result.country} is {result.city}\")\n",
+    "    print(f\"Population: {result.population}\")\n",
+    "except ValueError as e:\n",
+    "    print(f\"Validation error: {e}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example:\n",
+    "1. We define a `Capital` model using Pydantic, specifying the structure and types of the expected data.\n",
+    "2. We use a field validator to ensure the population is always positive.\n",
+    "3. The `response_model` parameter in the `@openai.call` decorator tells Mirascope to parse the LLM's output into this structure.\n",
+    "4. We can then access the structured data directly, with type checking and validation already performed.\n",
+    "\n",
+    "Response models are particularly useful when you need consistent, structured output from your LLM calls, especially for data-driven applications."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Dynamic Configuration & Chaining\n",
+    "\n",
+    "Dynamic configuration allows you to modify LLM call settings at runtime, while chaining enables you to create sequences of LLM calls.\n",
+    "\n",
+    "Here's an example of dynamic configuration:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@openai.call(\"gpt-4o-mini\")\n",
+    "@prompt_template(\"Translate the following to {language}: {text}\")\n",
+    "def translate(text: str, language: str):\n",
+    "    return {\"temperature\": 0.7 if language == \"French\" else 0.5}\n",
+    "\n",
+    "\n",
+    "response = translate(\"Hello, world!\", \"French\")\n",
+    "print(response.content)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example, we're dynamically setting the `temperature` parameter based on the target language.\n",
+    "\n",
+    "Now, let's look at an example of chaining:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@openai.call(\"gpt-4o-mini\")\n",
+    "@prompt_template(\"Summarize the following text: {text}\")\n",
+    "def summarize(text: str):\n",
+    "    pass\n",
+    "\n",
+    "\n",
+    "@openai.call(\"gpt-4o-mini\")\n",
+    "@prompt_template(\"Translate the following to French: {text}\")\n",
+    "def translate_to_french(text: str):\n",
+    "    pass\n",
+    "\n",
+    "\n",
+    "original_text = \"Recently, it's been really hot, so I’m thinking of gathering with friends to enjoy some champagne outdoors. I’m looking for something refreshing and light, especially a champagne that’s enjoyable even on a hot day. If possible, I’d love to know about any recommended champagne that’s perfect for relaxing with everyone.\"\n",
+    "summary = summarize(original_text)\n",
+    "french_summary = translate_to_french(summary.content)\n",
```

Comment:
while you certainly can chain like this, we recommend chaining using dynamic configuration because then the final `response.model_dump` will include the entire dump from the whole chain and not just this call (which we should cover in the learn docs).

useful here I think to then show that output.